utHostUserResponse.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+import os
+import sys
+import subprocess
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(dir_path+"/../../../")
+
+class utHostUserResponse():
+    """Reads the user response
+    """
+    
+    def __init__(self, log=None):
+        """
+        Initializes a configuration reader instance based on the parameters.
+
+        Args:
+            log (class, optional): Parent log class. Defaults to None.
+        """
+        self.log = log
+    
+    def getUserYNFromHost(self, query=""):
+        """
+        Reads Y/N user response to the query.
+
+        Args:
+            query (str, optional): Query to the user. Defaults to ""
+
+        Returns:
+            bool: returns the response
+        """
+        print(query)
```

Comment:
You shouldn't be using print anywhere, self.log for the module needs to be assigned to `stdout` when you create it, but if you just print it you won't log the result.

```self.log("Please Enter a Y/N repsonse")```